src/app/index.route.coffee

```diff
@@ -37,6 +37,21 @@ angular.module 'mnoEnterpriseAngular'
         templateUrl: 'app/views/company/company.html'
         controller: 'DashboardCompanyCtrl'
         controllerAs: 'vm'
+      .state 'logout',
```

Comment:
Did you just swap the routes here? I can't see where `MARKETPLACE_CONFIG` is used?